subsys/partition_manager/Kconfig

```diff
@@ -138,18 +138,6 @@ config PM_EXTERNAL_FLASH_BASE
 
 endif
 
-config BUILD_WITH_TFM
-	bool
-	select PM_SINGLE_IMAGE
-	select ENTROPY_CC3XX if SOC_FAMILY_NRF
-	select NORDIC_SECURITY_BACKEND if SOC_FAMILY_NRF
-	imply MBEDTLS_ENABLE_HEAP
-	select NRF_SECURITY_RNG
-	select MBEDTLS_PSA_CRYPTO_C
-	imply MBEDTLS_PSA_CRYPTO_STORAGE_C
-	select MBEDTLS_USE_PSA_CRYPTO
-	imply MBEDTLS_MD5_C
-
```

Comment:
please look at comments in dedicated commits so that this is not being introduced in the first place.

Reply:
This will be cleaned away, yes

Reply:
Fixed